### PR TITLE
Fix bug Alessio

### PR DESCRIPTION
**Pull request by AlessioVerardo**
In this PR, 
I made the id of the views more consistent with all the project. I also added some tests for the different hub fragment. 

(We must merge the first PR before this one)

### CODE REVIEW

File: app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/NavigationViewFragmentTest.kt [hunk 1]
```diff
@@ -63,33 +69,36 @@ class NavigationViewFragmentTest {
...
+
+        //Initial state
         Espresso.onView(withId(R.id.ic_home)).perform(click())
-        Espresso.onView(withId(R.id.id_fragment_home)).check(matches(isDisplayed()))
+        Espresso.onView(withId(R.id.id_fragment_home_visitor)).check(matches(isDisplayed()))
 
-        Espresso.onView(withId(R.id.nav_search)).perform(click())
-        Espresso.onView(withId(R.id.id_fragment_home)).check(matches(isDisplayed()))*/
+        //Espresso.onView(withId(R.id.ic_map)).perform(click())
+        //Espresso.onView(withId(R.id.id_fragment_map)).check(matches(isDisplayed()))
```
Comment (Mathematou) [inline app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/NavigationViewFragmentTest.kt:93]: Maybe we could remove these

File: app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/home/ActivityProvideFragmentTests.kt [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+import org.hamcrest.Matchers
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.anyOrNull
+import org.mockito.Mockito.`when` as When
+
+class ActivityProvideFragmentTests {
```
Comment (Feez2403) [inline app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/home/ActivityProvideFragmentTests.kt:38]: typo activityprovide

**Review by Mathematou — APPROVED** [review]
Nice refactor of the ids

**Review by Cell00phane — APPROVED** [review]
Nothing to say! Good refactoring and renaming the ids of the views to match the convention of the app. Good tests too!

**Review by Feez2403 — APPROVED** [review]
Nice refactor, nothing much to say :)

File: app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/home/ActivityProvideFragmentTests.kt (hunk 2 above)
Comment (AlessioVerardo) [reply app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/home/ActivityProvideFragmentTests.kt:38]: Yep !

File: app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/NavigationViewFragmentTest.kt (hunk 1 above)
Comment (AlessioVerardo) [reply app/src/androidTest/java/com/github/sdpteam15/polyevents/view/fragments/NavigationViewFragmentTest.kt:93]: Yes, sure !

**Pull request merged**